chart/templates/deployment.yaml

```diff
@@ -47,6 +47,11 @@ spec:
             - name: {{ $name }}
             {{- toYaml $config | nindent 14 }}
             {{- end }}
+            {{- if $.Values.customNetworkConfig }}
+            - name: custom-network-config-volume
+              mountPath: /app/networks-config.json
+              subPath: networks-config.json
+            {{- end }}
```

Comment:
We already have custom volumes and volume mounts available from the values.yml that is templatized. I think we should just move this there so it's always present and has a default config with all public networks.